### PR TITLE
RHIDP-2871: Update Backstage CR in TechDocs doc

### PR DESCRIPTION
**Pull request by linfraze**
**Version(s):**
1.2

**Issue:**
https://issues.redhat.com/browse/RHIDP-2871 

**Link to docs preview:**
Operator section: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-439/admin-rhdh/#proc-techdocs-configure-odf-operator_assembly-admin-templates 
Helm section: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-439/admin-rhdh/#proc-techdocs-configure-odf-helm_assembly-admin-templates 

**Reviews:**
- [ ] SME: @kadel 
- [ ] QE: TBD
- [ ] Docs review: TBD

### CODE REVIEW

File: modules/getting-started/proc-techdocs-configure-odf-helm.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [id="proc-techdocs-configure-odf-helm_{context}"]
 = Making object storage accessible to containers by using the Helm chart
 
-Creating a `ObjectBucketClaim` custom resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Helm chart configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
+Creating a `Backstage` custom resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Helm chart configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
```
Comment (kadel) [inline modules/getting-started/proc-techdocs-configure-odf-helm.adoc:5]: This should remain as `ObjectBucketClaim`. It was correct before. 

```suggestion
Creating a `ObjectBucketClaim ` custom resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Helm chart configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
```

File: modules/getting-started/proc-techdocs-configure-odf-operator.adoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 [id="proc-techdocs-configure-odf-operator_{context}"]
 = Making object storage accessible to containers by using the Operator
 
-Creating a `ObjectBucketClaim` Custom Resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Operator configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
```
Comment (kadel) [inline modules/getting-started/proc-techdocs-configure-odf-operator.adoc:5]: This should remain as `ObjectBucketClaim`. It was correct before. 

```suggestion
Creating a `ObjectBucketClaim ` custom resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Helm chart configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
```

File: modules/getting-started/proc-techdocs-configure-odf-operator.adoc [hunk 3]
```diff
@@ -21,10 +21,14 @@ These variables are then used in the TechDocs plugin configuration.
 
 .Procedure
 
-* In the {product-short} `ObjectBucketClaim` CR, enter the name of the {product-short} `ObjectBucketClaim` config map as the value for the `spec.application.extraEnvs.configMaps` field and enter the {product-short} `ObjectBucketClaim` secret name as the value for the `spec.application.extraEnvs.secrets` field. For example:
+* In the {product-short} `Backstage` CR, enter the name of the {product-short} `ObjectBucketClaim` config map as the value for the `spec.application.extraEnvs.configMaps` field and enter the {product-short} `ObjectBucketClaim` secret name as the value for the `spec.application.extraEnvs.secrets` field. For example:
```
Comment (kadel) [inline modules/getting-started/proc-techdocs-configure-odf-operator.adoc:24]: This change is **OK**. Here, it was wrong before.

File: modules/getting-started/proc-techdocs-configure-odf-operator.adoc [hunk 4]
```diff
@@ -2,7 +2,7 @@
 [id="proc-techdocs-configure-odf-operator_{context}"]
 = Making object storage accessible to containers by using the Operator
 
-Creating a `ObjectBucketClaim` Custom Resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Operator configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
+Creating a `Backstage` Custom Resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Operator configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
```
Comment (kadel) [inline modules/getting-started/proc-techdocs-configure-odf-operator.adoc:5]: This should remain as `ObjectBucketClaim`. It was correct before. 

```suggestion
Creating a `ObjectBucketClaim` Custom Resource (CR) automatically generates both the {product-short} `ObjectBucketClaim` config map and secret. The config map and secret contain `ObjectBucket` access information. Adding the access information to the Operator configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
```

**Review by Gerry-Forde — APPROVED** [review]
/lgtm

**Comment by Gerry-Forde** [comment]
@kadel I'm picking up this PR that @linfraze had created and you reviewed.

After applying your suggestions, I just wanted to check that your comment in the Jira has been applied (I don't think it has).

Could you please confirm if this section still needs to be updated per your Jira comment:
Screenshot 2024-08-20 at 12 29 21

**Comment by kadel** [comment]
> Could you please confirm if this section still needs to be updated per your Jira comment:

The section that you mentioned and showed in the screenshot is correct. 



> After applying your suggestions, I just wanted to check that your [comment in the Jira](https://issues.redhat.com/browse/RHIDP-2871?focusedId=25270524&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25270524) has been applied (I don't think it has).

This Jira issue talks about this:

Screenshot 2024-08-26 at 16 01 54



And TBH, I got a little bit lost there :-/ 
Now,  when I'm reading it it looks ok. In the highlighted block there should NOT be any `kind` as it is not Kubernetes Custom Resource, but snippet from Helm Chart configuration (values)


I think it looks ok NOW
/lgtm
/approve

**Review by kadel — APPROVED** [review]

**Pull request merged**